### ISSUE TITLE
No reader engine for this file or invalid file

### ISSUE THREAD

**Issue by DynamoRocket** [koreader/koreader#12430]
* KOReader version: 2024.07
* Device: Android 10 Tablet

#### Issue
When opening some CBZ files, I get the error in the title. I can extract the file and the images appear.

#### Steps to reproduce
Open CBZ file

**Comment by benoit-pierre** [comment]
Can provide some logcat logs? You can use [this python script](https://github.com/koreader/koreader/raw/master/tools/logcat.py) to filter out the relevant traces:
- enable debug logging
- quit koreader
- run `./logcat.py -c | tee crash.log`
- start koreader and reproduce your issue
- upload `crash.log` here

**Comment by DynamoRocket** [comment]
Its not really crashing, just failing to open some files, but here is what I have after that command

```
Traceback (most recent call last):
  File "/home//Downloads/./logcat.py", line 346, in <module>
    main()
  File "/home//Downloads/./logcat.py", line 337, in main
    subprocess.check_call(('adb', 'logcat', '-c'))
  File "/usr/lib64/python3.11/subprocess.py", line 408, in check_call
    retcode = call(*popenargs, **kwargs)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.11/subprocess.py", line 389, in call
    with Popen(*popenargs, **kwargs) as p:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.11/subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib64/python3.11/subprocess.py", line 1955, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'adb'
```

**Comment by benoit-pierre** [comment]
You do need to have ADB installed.

**Comment by benoit-pierre** [comment]
It's only some CBZ files? Not all?

**Comment by benoit-pierre** [comment]
Does [this version](https://ufile.io/xi4gxr0d) work?

Note: please make sure to uninstall the previous version before installing this one (KOReader does not store its settings in the application standard directory, so they should be preserved after uninstall).

**Comment by DynamoRocket** [comment]
That did work

**Comment by benoit-pierre** [comment]
Great! Thanks for testing.

**Comment by benoit-pierre** [comment]
Fixed by koreader/koreader-base#1917 via #12451.
